modules/update-common-terms.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+
+= Common Terms
+
+Control Plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines. The cluster itself manages all updates to the machines by the actions of the Cluster Version Operator (CVO), the Machine Config Operator (MCO), and a set of individual Operators.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OpenShift Update Service to view valid updates and update paths based on your current component versions and information in the graph.
+
+Machine Config Operator:: The _Machine Config Operator_ (MCO) is a cluster-level operator that manages the operating system and keeps the cluster up to date and configured. Through the MCO, platform administrators can configure and update systemd, CRI-O and Kubelet, the kernel, NetworkManager, and other system features on the nodes.
```

Comment:
nit: There are cluster-configs that are not managed by the MCO, e.g. ingress configurations, image-registry configurations, etc., so "keeps the cluster up to date and configured" -> "keeps the cluster machine configurations up to date" or "manages the operating system and machine configurations" or some such?